toolkit/tools/imagegen/configuration/packagerepo.go

```diff
@@ -25,15 +25,30 @@ import (
 // repository configuration will be saved in the installed system if specified, and only
 // available during the installation process if not
 type PackageRepo struct {
-	Name    string `json:"Name"`
-	BaseUrl string `json:"BaseUrl"`
-	Install bool   `json:"Install"`
+	Name     string `json:"Name"`
+	BaseUrl  string `json:"BaseUrl"`
+	Install  bool   `json:"Install"`
+	GPGCheck bool   `json:"GPGCheck"` // Default value is true
+	GPGKeys  string `json:"GPGKeys"`  // Default value is "file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY"
+}
+
+func getDefaultGPGCheck() bool {
+	return true
+}
+
+func getDefaultGPGKeys() string {
+	return "file:///etc/pki/rpm-gpg/MICROSOFT-RPM-GPG-KEY file:///etc/pki/rpm-gpg/MICROSOFT-METADATA-GPG-KEY"
 }
 
 // UnmarshalJSON Unmarshals a PackageRepo entry
 func (p *PackageRepo) UnmarshalJSON(b []byte) (err error) {
 	// Use an intermediate type which will use the default JSON unmarshal implementation
 	type IntermediateTypePackageRepo PackageRepo
+
+	// Set default values
+	p.GPGCheck = getDefaultGPGCheck()
+	p.GPGKeys = getDefaultGPGKeys()
```

Comment:
Since this is the only place we use these two new functions, I think we can simplify to use `const` instead. This way we'd be also more consistent with how we use constants in other places of the code.

Reply:
There are a few spots in these files we do make these types of defaults publicly available, was just following that flow. Just using consts is better here though, agreed.